common-custom-user-data-maven-extension/pom.xml

```diff
@@ -65,7 +65,7 @@
         <dependency>
             <groupId>com.gradle</groupId>
             <artifactId>gradle-enterprise-maven-extension</artifactId>
-            <version>1.6.4</version>
+            <version>1.7-branch-francois_6769_maven_build_cache_api-prerelease</version>
```

Comment:
This is what I used locally, and will obviously have to change before release

Reply:
Used `1.6.5` for the upcoming release.